tests/serial/time/implicit/serial-coupling/WaveformSubsyclingWithConstantAcceleration.cpp

```diff
@@ -0,0 +1,135 @@
+#ifndef PRECICE_NO_MPI
+
+#include <math.h>
+#include <precice/precice.hpp>
+#include <vector>
+#include "testing/Testing.hpp"
+
+using namespace precice;
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(Time)
+BOOST_AUTO_TEST_SUITE(Implicit)
+BOOST_AUTO_TEST_SUITE(SerialCoupling)
+
+/**
+ * @brief Simple test to ensure that underrelaxation is applied to every  timestep.
+ */
+BOOST_AUTO_TEST_CASE(WaveformSubsyclingWithConstantAcceleration)
+{
+  PRECICE_TEST("SolverOne"_on(1_rank), "SolverTwo"_on(1_rank));
+
+  Participant precice(context.name, context.config(), 0, 1);
+
+  MeshID meshID;
+  DataID writeDataID;
+  DataID readDataID;
+
+  typedef double (*DataFunction)(double);
+
+  DataFunction dataOneFunction = [](double t) -> double {
+    return (double) (2 * t);
+  };
+  DataFunction dataTwoFunction = [](double t) -> double {
+    return (double) (3 * t);
+  };
+  DataFunction writeFunction;
+  DataFunction readFunction;
+
+  std::string meshName, writeDataName, readDataName;
+  if (context.isNamed("SolverOne")) {
+    meshName      = "MeshOne";
+    writeDataName = "DataOne";
+    writeFunction = dataOneFunction;
+    readDataName  = "DataTwo";
+    readFunction  = dataTwoFunction;
+  } else {
+    BOOST_TEST(context.isNamed("SolverTwo"));
+    meshName      = "MeshTwo";
+    writeDataName = "DataTwo";
+    writeFunction = dataTwoFunction;
+    readDataName  = "DataOne";
+    readFunction  = dataOneFunction;
+  }
+
+  double   writeData = 0;
+  double   readData  = 0;
+  double   v0[]      = {0, 0, 0};
+  VertexID vertexID;
+  vertexID = precice.setMeshVertex(meshName, v0);
+
+  int    nSubsteps          = 7; // perform subcycling on solvers. 4 steps happen in each window.
```

Comment:
```suggestion
  int    nSubsteps          = 7; // perform subcycling on solvers. 7 steps happen in each window.
```
Is there a specific reason for using 7 steps? Should 2 steps not already suffice?